test/e2e/storage/csi_volumes.go

```diff
@@ -205,13 +213,13 @@ var _ = utils.SIGDescribe("CSI Volumes", func() {
 	})
 })
 
-func createCSIDriver(csics csiclient.Interface, attachable bool) *csi.CSIDriver {
+func createCSIDriver(csics csiclient.Interface, attachable bool) *csiv1alpha1.CSIDriver {
```

Comment:
this function name is overloaded with the `csiTestDriver` Interface definition in the same file. I don't think its doing the same thing, can we change the name or fold it into the hostPath driver?

Reply:
Yeah I'll be updating the e2e test again in a couple of days and could include this in that PR.